### PR TITLE
Run integration tests on `poco-chain@native-v5.4.2-5s`

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
```
Comment (jeremyjams) [inline CHANGELOG.md:2]: `TODO: Bump version`

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ chain:
   id: ${IEXEC_BLOCKCHAIN_ADAPTER_API_CHAIN_ID:65535}
   node-address: ${IEXEC_BLOCKCHAIN_ADAPTER_API_NODE_ADDRESS:http://localhost:8545} #https://viviani.iex.ec/
   block-time: ${IEXEC_BLOCKCHAIN_ADAPTER_API_BLOCK_TIME:1} #in seconds, use 15 for mainnet
-  hub-address: ${IEXEC_BLOCKCHAIN_ADAPTER_API_HUB_ADDRESS:0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002} #0x3eca1B216A7DF1C7689aEb259fFB83ADFB894E7f
+  hub-address: ${IEXEC_BLOCKCHAIN_ADAPTER_API_HUB_ADDRESS:0xC129e7917b7c7DeDfAa5Fff1FB18d5D7050fE8ca} #0x3eca1B216A7DF1C7689aEb259fFB83ADFB894E7f
```
Comment (jeremyjams) [inline src/main/resources/application.yml:19]: 1. Leave conf as is was and only update itest values
or
2. Update default `block-time` & `is-sidechain` in this hotfix as well

File: src/itest/java/com/iexec/blockchain/IntegrationTests.java [hunk 3]
```diff
@@ -51,7 +51,12 @@ class IntegrationTests {
 
     public static final String USER = "admin";
     public static final String PASSWORD = "whatever";
-    public static final int BLOCK_TIME_MS = 1000;
+    public static final int BLOCK_TIME_MS = 5000;
+    public static final int MAX_BLOCK_TO_WAIT = 3;
+    public static final int POLLING_PER_BLOCK = 2;
+    public static final int POLLING_TIME_MS = BLOCK_TIME_MS / POLLING_PER_BLOCK;
```
[the remark this answers is not in the document]
Comment (jeremyjams) [reply src/itest/java/com/iexec/blockchain/IntegrationTests.java]: Updated https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/pull/81/commits/fd94486c8a91afe7e3c6f6d3aa3d5a515d24af2b :+1:

File: src/itest/java/com/iexec/blockchain/IntegrationTests.java [hunk 4]
```diff
@@ -144,20 +152,22 @@ public void shouldBurstTransactionsWithAverageOfOneTxPerBlock(){
     }
 
     private String triggerDeal(int taskVolume) {
+        int secondsPollingInterval = POLLING_TIME_MS / 1000;
+        int secondsTimeout = secondsPollingInterval * MAX_POLLING_ATTEMPTS;
```
[the remark this answers is not in the document]
Comment (jeremyjams) [reply src/itest/java/com/iexec/blockchain/IntegrationTests.java]: Yes, see https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/pull/81/commits/46ecbfb85b9e600954bdd7a8ca7088371d9f1e52 :)

[the remark this answers is not in the document]
Comment (mcornaton) [reply src/itest/java/com/iexec/blockchain/IntegrationTests.java]: Beware:
```java
int secondsPollingInterval = Duration.ofMillis(POLLING_INTERVAL_MS)
    .toSecondsPart();
```
probably does not what you're expecting it to do. It get the seconds part of the duration. E.g, `Duration.ofMillis(61000).toSecondsPart()` return `1`.
BTW, I was talking about having `Duration pollingInterval` and `Duration timeout` parameters in our `createXxx` methods. Not creating `Duration` objects that are immediately thrown away.

[the remark this answers is not in the document]
Comment (mcornaton) [reply src/itest/java/com/iexec/blockchain/IntegrationTests.java]: Exactly!

**Review by mcornaton — APPROVED** [review]

**Pull request merged**